src/palace/manager/api/admin/controller/catalog_services.py

```diff
@@ -21,7 +21,7 @@
 
 
 class CatalogServicesController(
-    IntegrationSettingsController[MARCExporter],
+    IntegrationSettingsController[MarcExporter],
```

Comment:
I'm partial to lower-casing acronyms when using CamelCase, giving `MarcExporter` instead of `MARCExporter`, so we don't end up with abominations like `MARCXMLExporter`. This change was unintentional in my refactoring however, so I'm open to the argument that it should be changed back to `MARCExporter`.